core/src/main/scala/kafka/tools/DumpLogSegments.scala

```diff
@@ -280,7 +284,7 @@ object DumpLogSegments {
               if (batch.magic >= RecordBatch.MAGIC_VALUE_V2) {
                 print(" sequence: " + record.sequence + " headerKeys: " + record.headers.map(_.key).mkString("[", ",", "]"))
               } else {
-                print(s" crc: ${record.checksumOrNull} isvalid: ${record.isValid}")
+                print(s" crc: $checksum isvalid: ${record.isValid}")
```

Comment:
Are we printing the checksum twice or am I missing something?

Reply:
yep, it printed the checksum twice if magic code is smaller than 2. Will remove the duplicate print.